packages/gatsby-plugin-page-creator/src/create-page-wrapper.ts

```diff
@@ -4,6 +4,7 @@ import { createClientOnlyPage } from "./create-client-only-page"
 import { createPagesFromCollectionBuilder } from "./create-pages-from-collection-builder"
 import systemPath from "path"
 import { trackFeatureIsUsed } from "gatsby-telemetry"
+import { Reporter } from "gatsby"
```

Comment:
Should this be a type only import?

Reply:
As discussed, nope because we decided not to use `import type`.